app/store/ocean.ts

```diff
@@ -3,7 +3,7 @@ import { createSelector, createSlice, PayloadAction } from '@reduxjs/toolkit'
 
 export interface OceanTransaction {
   broadcasted: boolean
-  signed: CTransactionSegWit
+  signer: () => Promise<CTransactionSegWit>
```

Comment:
Major change here @thedoublejay.
I moved the signing logic to be executed inside OceanInterface component. The signing itself is taking way longer than the broadcast, the UI looks frozen before the dispatch(queue) happened and kinda defeat the purpose of having this drawer.

Reply:
yup good idea